src/board_controller/neuropawn/knight.cpp

```diff
@@ -0,0 +1,328 @@
+#include <math.h>
+#include <string.h>
+#include <vector>
+
+#include "custom_cast.h"
+#include "knight.h"
+#include "serial.h"
+#include "timestamp.h"
+
+constexpr int Knight::start_byte;
+constexpr int Knight::end_byte;
+
+Knight::Knight (int board_id, struct BrainFlowInputParams params) : Board (board_id, params)
+{
+    serial = NULL;
+    is_streaming = false;
+    keep_alive = false;
+    initialized = false;
+    if (board_id == (int)BoardIds::NEUROPAWN_KNIGHT_BOARD)
+    {
+        min_package_size = 21;
```

Comment:
Looks like this if is not needed, there is just 1 board, so it will always be 21. Unless you are going to add more boards using this class